keras/layers/embeddings.py

```diff
@@ -78,14 +78,14 @@ def __init__(self, input_dim, output_dim,
                  embeddings_constraint=None,
                  mask_zero=False,
                  input_length=None,
+                 sparse_grad=False,
```

Comment:
Add this in doc string explaining the usage and note it's only for mxnet backend